transports/rabbitmq/cluster.md

```diff
@@ -1,20 +1,15 @@
 ---
 title: In a cluster
 component: Rabbit
-reviewed: 2017-01-13
+reviewed: 2018-11-22
 related:
  - nservicebus/operations
 redirects:
  - nservicebus/rabbitmq/cluster
 ---
 
-The [RabbitMQ Clustering Guide](https://www.rabbitmq.com/clustering.html) recommends that client applications should not hard-code the hostnames or IP addresses of the machines hosting nodes in a RabbitMQ cluster. An approach such as DNS or load balancing should be used instead. This approach is recommended when using the NServiceBus RabbitMQ transport.
+The [RabbitMQ Clustering Guide](https://www.rabbitmq.com/clustering.html#clients) recommends that client applications should not hard-code the hostnames or IP addresses of the machines hosting nodes in a RabbitMQ cluster. An approach such as DNS or load balancing should be used instead. This approach is recommended when using the NServiceBus RabbitMQ transport.
```

Comment:
With my editor hat on: Now that the hyperlink does not point to the whole guide, it's semantically incorrect, but TBH I can't think of a good way to structure this sentence which would give us a better option. It's only a minor point anyway.